src/less/pagination.less

```diff
@@ -45,43 +45,32 @@
   > li > a,
   > li > span {
     &:hover {
-      color: @pagination-hover-color;
       background-color: @pagination-hover-bg;
-      border-color: @pagination-hover-border;
     }
   }
 
   > .active > a,
   > .active > span {
-
-		color: @pagination-active-color;
 		cursor: default;
 		background-color: @pagination-active-bg;
-		border-color: @pagination-active-border;
 		font-weight: @font-weight-bold;
 
     &:hover {
-      color: @pagination-active-color;
-      cursor: default;
-      background-color: @pagination-active-bg;
-      border-color: @pagination-active-border;
+      background-color: @pagination-active-bg;  
     }
   }
 
   > .disabled {
     > span,
     > a {
-
-			color: @pagination-disabled-color;
+      color: @black;
+      background-color: @body-bg;
```

Comment:
disabled background-color should be @pagination-bg and the background shouldn't change on hover